endpoints-framework/src/main/java/com/google/api/server/spi/config/model/ApiIssuerConfigs.java

```diff
@@ -13,6 +13,9 @@
   public static final IssuerConfig GOOGLE_ID_TOKEN_ISSUER = new IssuerConfig(
       Constant.GOOGLE_ID_TOKEN_NAME, "accounts.google.com",
       "https://www.googleapis.com/oauth2/v1/certs");
+  public static final IssuerConfig GOOGLE_ID_TOKEN_ISSUER_ALT = new IssuerConfig(
+      Constant.GOOGLE_ID_TOKEN_NAME + "_alt", "https://accounts.google.com",
```

Comment:
Can we define another constant in Constant.java instead of doing string concatenation everywhere?

Also I'd prefer to suffixing it with "_https" instead of "_alt" to be more explicit.